src/java/org/apache/cassandra/tools/SSTableExpiredBlockers.java

```diff
@@ -122,12 +135,37 @@ public static Multimap<SSTableReader, SSTableReader> checkForExpiredSSTableBlock
         return blockers;
     }
 
-    private static String formatForExpiryTracing(Iterable<SSTableReader> sstables)
+    @VisibleForTesting
+    public static String formatForExpiryTracing(boolean humanReadable, Iterable<SSTableReader> sstables)
     {
         StringBuilder sb = new StringBuilder();
 
         for (SSTableReader sstable : sstables)
-            sb.append(String.format("[%s (minTS = %d, maxTS = %d, maxLDT = %d)]", sstable, sstable.getMinTimestamp(), sstable.getMaxTimestamp(), sstable.getMaxLocalDeletionTime())).append(", ");
+        {
+            long minTimestamp = sstable.getMinTimestamp();
+            long maxTimestamp = sstable.getMaxTimestamp();
+            long maxLocalDeletionTime = sstable.getMaxLocalDeletionTime();
+            if (humanReadable)
+            {
+                sb.append(format("[%s (minTS = %s, maxTS = %s, maxLDT = %s, diskSize = %s)]",
```

Comment:
Nit: I think extracting this string for the format will help with maintenance in the future, having to change it only once instead of here and on line 160.